ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -366,14 +374,9 @@ objects:
           mkdir -p /tmp/artifacts/pods /tmp/artifacts/nodes /tmp/artifacts/metrics /tmp/artifacts/bootstrap /tmp/artifacts/network
 
 
-          if [ -f /tmp/artifacts/installer/terraform.tfstate ]
+          if [ -f /tmp/artifacts/terraform/terraform.tfstate ]
```

Comment:
Everyone else puts this under the installer directory:
```console
$ git grep -h terraform.tfstate origin/master -- ci-operator/templates/openshift/installer | grep -o '/tmp.*tfstate' | sort | uniq -c
     12 /tmp/artifacts/installer/terraform.tfstate
```
but I guess that's because they're running Terraform via the installer.  So +1 to this, although I was initially confused ;).